### ISSUE TITLE
output in nexus file

### ISSUE THREAD

**Issue by MarliesJFrancine** [NicolaDM/MAPLE#24]
Is it possible to convert the output file from .tree format to a .nexus format? Several programs for further analysis require the file to be in nexus format. However, standard methods such as the write.nexus in R (ape package) do not seem to work because it cannot recognize the .tree file format.

**Comment by NicolaDM** [comment]
Yes, this is already implemented: MAPLE will write a nexus output tree when necessary, that is when using option --estimateMAT (which estimates a mutation-annotated tree) or --aBayesPlus (which calculates branch supports). Is this enough for you or would you need nexus trees without these additional bits of information?

**Comment by MarliesJFrancine** [comment]
Thanks for the quick reply. I was looking for a nexus file without that additional information :)

**Comment by NicolaDM** [comment]
If you don't want to wait for me to implement this, a quick solution is to ask MAPLE to create a NEXUS tree (for example using option --estimateMAT ), and then copy-paste the output newick tree into the output nexus format, replacing the original tree in the nexus file. Does this work for you?

Also, do you know what is the problem with the trees in .tree format? These are standard newick trees, every software should be able to read them. Have you tried changing the extensions of the files?

**Comment by MarliesJFrancine** [comment]
I will apply the quick solution for now, but thank you for implementing it perhaps at a later point :)

I unfortunately do not know what the issue is exactly, even when changing the extensions, the software says "input file is not in nexus format", so perhaps there is a slight difference somewhere between the two files.
